reinforcement_learning/bindings/cs/rl.net.native/rl.net.config.h

```diff
@@ -0,0 +1,17 @@
+#include "rl.net.native.h"
+
+#pragma once
+
+//int load_file(const std::string& file_name, std::string& file_data);
```

Comment:
[nit] do you plan to remove these comments?

Reply:
Yes